tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -33,48 +32,71 @@
 from gcs_bucket_mover import sts_job_status
 
 _CHECKMARK = u'\u2713'.encode('utf8')
+_LOGGER = None
 
 
-def move_bucket(conf):
+def move_bucket(conf):  # Majority statements are logging. pylint: disable=too-many-statements
     """Main entry point for the bucket mover script
 
     Args:
-        conf: the argparser parsing of command line options
+        conf: the configargparser parsing of command line options
     """
 
     # Load the environment config values set in config.sh and create the storage clients.
     config = configuration.Configuration.from_conf(conf)
 
-    print('Using the following service accounts for GCS credentials: ')
-    print('Source Project - {}'.format(
+    # We need to create the cloud logging client before we can set the global logger.
+    global _LOGGER  # pylint: disable=global-statement
```

Comment:
The global statement warning is there for a reason. You should define _LOGGER outside this function (and remove the `global` statement). In fact since this is in all caps this is considered a constant and it should be defined and set at the global scope and never changed.